src/extension/ai/stream.ts

```diff
@@ -20,7 +32,7 @@ export interface CompletionHandlers {
   // If null is returned then no request is generated
   buildRequest: (
     cellChangeEvent: CellChangeEvent,
-    firstRequest: boolean,
+    options: { firstRequest: boolean; handleNewCells: boolean },
```

Comment:
Can you explain this change? What is the meaning of handleNewCells?

Reply:
It's hacky and not final yet, I introduced it to be able to reuse the `buildRequest` method to control my debounce logic. My plan was to rewrite once I have both the debounce and the tests working end-to-end. I'll let you know when it's ready for a final review.

Reply:
@jlewi reverse this change as part of a refactor for clarity